src/__tests__/setup.ts

```diff
@@ -3,7 +3,7 @@
 
 // type-only import, because in runtime it tries to access Jest globals, which do not exist
```

Comment:
I checked through files in this PR, and it's the only place where there was a comment before an import. Here it's just 2 lines, so doesn't really matter if the comment was left behind or not. But it could be the case in the other PRs that the comment was left behind an now it's kinda difficult to get what it's referring to. Just FYI